source/2019-09-20-ember-3-13-released.md

```diff
@@ -0,0 +1,150 @@
+---
+title: Ember 3.13 Released
+author: Kenneth Larsen
+tags: Releases, 2019, 3, 3.13
+responsive: true
+---
+
+Today the Ember project is releasing version 3.13 of Ember.js, Ember Data, and Ember CLI. 
+
+Additionally, versions 3.12 of Ember and Ember Data are now promoted to LTS, which stands for Long Term Support. An LTS version of Ember continues to receive security updates for 9 release cycles (54 weeks) and bugfixes for 6 cycles (36 weeks). LTS releases typically occur every four minor versions. The previous LTS versions for Ember and Ember Data were 3.4.
+
+If you are upgrading from the previous LTS version to 3.12 then you are getting access to these features:
+
+[3.6](https://blog.emberjs.com/2018/12/13/ember-3-6-released.html)
+* `router` service 
+* Support for native classes
+
+[3.8](https://blog.emberjs.com/2019/02/27/ember-3-8-released.html)
+* The array helper
+* Element Modifier Manager
+
+[3.10](https://blog.emberjs.com/2019/05/21/ember-3-10-released.html)
+* Nested angle bracket component invocation
+* Angle bracket invocation for built-in components
+* Support for native decorators
+
+[3.11](https://blog.emberjs.com/2019/07/15/ember-3-11-released.html)
+* Forwarding Element Modifiers with `...attributes`
+* The `{{fn}}` and `{{on}}` modifiers
+
+This release kicks off the 3.14 beta cycle for all sub-projects. We encourage our community (especially addon authors) to help test these beta builds and report any bugs before they are published as a final release in six weeks' time. The [ember-try](https://github.com/ember-cli/ember-try) addon is a great way to continuously test your projects against the latest Ember releases.
+
+You can read more about our general release process here:
+
+- [Release Dashboard](http://emberjs.com/builds/)
+- [The Ember Release Cycle](http://emberjs.com/blog/2013/09/06/new-ember-release-process.html)
+- [The Ember Project](http://emberjs.com/blog/2015/06/16/ember-project-at-2-0.html)
+- [Ember LTS Releases](http://emberjs.com/blog/2016/02/25/announcing-embers-first-lts.html)
+
+---
+
+## Ember.js
+
+Ember.js is the core framework for building ambitious web applications.
+
+### Changes in Ember.js 3.13
+
+Ember.js 3.13 is an incremental, backwards compatible release of Ember with bugfixes, performance improvements, and minor deprecations. There are five (5) new features, one (1) deprecation, and several bugfixes in this version.
+
+#### New Features (5)
+
+**Tracked Properties and Tracked Property Updates (1 of 5)**
```

Comment:
Suggestion for semantics: these are being **strongly emphasized** but what you really mean is "This is a section." These should be `#####` for `h5` tags. We need to add styles for that to look right, which I can certainly do if someone points me to where (and I can go back and update all previous blog posts using this format as well).

Reply:
You're right. It's been there because we don't have the styles for it, I believe. There is a redesign going on right now which I hope will resolve the issue